WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -97,3 +135,23 @@ extension SiteAddressViewController: NUXKeyboardResponder {
         keyboardWillHide(notification)
     }
 }
+
+
+// MARK: - Constants
+extension SiteAddressViewController {
+    /// Rows listed in the order they were created
+    ///
+    enum Row {
```

Comment:
There is only 1 section for each screen so I left Sections out of the enum. I can add it here if it helps.

Reply:
I don't think that's necessary.